sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/Clauses/FromClause.cs

```diff
@@ -12,6 +10,8 @@ namespace Azure.DigitalTwins.Core.QueryBuilder
     /// </summary>
     internal class FromClause : BaseClause
     {
+        private string _collection;
```

Comment:
Not sure how valuable this private field would be if it is left as is, I think you might be able to translate the Enum value into a string and keep it in here as a property. 
so: 
the `string _collection` can be a property `public string Collection {get; private set;}` and you can translate the enum value into the string representation also.

Reply:
Fair point. I'll change just the setter method to private, but I don't understand why we'd want to translate the enum to a string since this parameter is meant to be used in a overloaded From clause constructor where the user overrides the enum and injects a straight string for the collection type instead.

Reply:
As we discussed offline, You can store the provided collection in a string format by calling ToString() on the enum value provided in the constructor.